sched/group/group_setuptaskfiles.c

```diff
@@ -81,13 +81,15 @@ int group_setuptaskfiles(FAR struct task_tcb_s *tcb,
 
   /* Duplicate the parent task's file descriptors */
```

Comment:
@Donny9 please update this comment to reflect the new code

Reply:
Done~